flow/table.go

```diff
@@ -268,22 +531,24 @@ func (ft *Table) update(updateFrom, updateTime int64) {
 	logging.GetLogger().Debugf("flow table update: %d, %d", updateFrom, updateTime)
 
 	var updatedFlows []*Flow
-	for k, f := range ft.table {
+	for _, k := range ft.table.Keys() {
```

Comment:
This will create a copy of all the keys. It would be nice to have an implementation allowing to iterate over the values/keys

Reply:
right now this library doesn't expose such a functionality, would you like me to override it? is it ok to do so (there are the licence issues or whatever). do you want us to open a PR?

Reply:
I need to check what the cost of this. Yes maybe opening a PR.